modules/data.atmosphere/vignettes/tdm_downscaling.Rmd

```diff
@@ -0,0 +1,55 @@
+Temporally Downscale Meteorology
+===============
+
+
+### Subdaily Training Data
+
+Examples:
+
+* Fluxnet2015 30-minute/hourly
+* Ameriflux 30-minute/hourly
+
+### Extract Training Data and Merge All Years Into 1 File 
+
+```{r}
+library(PEcAn.data.atmosphere)
+library(PEcAn.DB)
+download.Fluxnet2015(sitename = 'Niwot Ridge Forest/LTER NWT1 (US-NR1)', username = 'pecan', overwrite = FALSE, outfolder = '~/Example', 
+   start_date = '2000/01/01', end_date = '2014/12/31')
```

Comment:
note: in this case by "outfolder's root folder" I mean defining ```outfolder <- "~/Example/"```, adding an explicit ```dir.create``` to make sure it exists, and then using file.path to create step-specific in and outdirs on the fly (e.g. in nc.merge it'd be ```in.path = file.path(outfolder,"gapfill")``` and ```outfolder=file.path(outfolder,"training_data")```. No need to include "NR1_" in the name of the training data folder.